README.md

```diff
@@ -194,12 +194,13 @@ The rule assumes some sane defaults:
 
 In order to speed up testing, Barista keeps in mind some considerations.
 - **Scrolls when needed**: Interacting with Espresso in a `ScrollView` requires you to scroll to each view, which sometimes doesn't work the first time. Also trying to scroll outside a `ScrollView` produces an `Exception`, forcing you to change the test depending on the layout. To keep tests simpler, Barista scrolls automatically before interacting with any `View`, and only does it if needed.
-- **Just interacts with displayed Views**: Interacting with `View`s inside a `ViewPager` throws `AmbiguousViewMatcherException`, cos the views you interact with will be potentially repeated on different pages. Barista only interacts with displayed widgets, so you can focus on the behavior instead of wasting time on details.  
+- **Scrolls on all views**: Barista scrolls on all scrollable views, including `NestedScrollView`
```

Comment:
It misses the period. In addition, what do you think about my previous suggestion? You fully ignored it 😞 . It was hard for me to write it 😂